### PR TITLE
library: Add Menu entry

### LINKED ISSUE #321: Add Menu library entry

### PR DESCRIPTION
**Pull request by AkshayWarrier**
Closes #321

### CODE REVIEW

File: src/Library/demos/Menu/main.blp [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  }
+
+  section {
+    label: _("Settings");
+
+    submenu {
+      label: _("Appearance");
+
+      item {
+        label: _("Force System Default");
```
Comment (andyholmes) [inline src/Library/demos/Menu/main.blp:93]:
```suggestion
        label: _("System Default");
```

File: src/Library/demos/Menu/main.blp [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    action: "text.bold";
+  }
+
+  submenu {
+    label: _("Font Color");
+
+    item {
+      label: _("White");
+      action: "text.color";
+      target: "white";
```
Comment (andyholmes) [inline src/Library/demos/Menu/main.blp:144]:
```suggestion
      label: _("Black");
      action: "text.color";
      target: "black";
```

Let's make the default `black`. The default makes it go nearly invisible, and I think that's a bit confusing.

**Review by andyholmes — CHANGES_REQUESTED** [review]
Nice one.

A few teaks and I think this is good to go.

**Comment by sonnyp** [comment]
@AkshayWarrier ping :)

File: src/Library/demos/Menu/main.blp (hunk 2 above)
Comment (AkshayWarrier) [reply src/Library/demos/Menu/main.blp:144]: Sorry for the late response 😅 

But I was running Workbench in dark mode so I completely forgot that white text would go invisible in light mode, maybe we could choose a color that's neither black nor white?

Maybe gray?

image
image

We could choose any other random color

Comment (andyholmes) [reply src/Library/demos/Menu/main.blp:144]: How about just `green`? I hadn't thought of dark mode myself, so red/green/blue seems safe and legible for all variants.

**Review by andyholmes — CHANGES_REQUESTED** [review]

**Review by andyholmes — APPROVED** [review]
No need to wait for a simple fix, since you were up late trying to get Workbench to build.

**Comment by AkshayWarrier** [comment]
Thank you :)

**Pull request merged**